src/server/src/main/java/io/cassandrareaper/jmx/JmxConnectionFactory.java

```diff
@@ -19,26 +19,36 @@
 import io.cassandrareaper.core.Cluster;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.Callable;
 import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeoutException;
 import java.util.concurrent.atomic.AtomicInteger;
 
 import com.codahale.metrics.MetricRegistry;
 import com.datastax.driver.core.policies.EC2MultiRegionAddressTranslator;
 import com.google.common.base.Optional;
 import com.google.common.collect.Maps;
+import jersey.repackaged.com.google.common.collect.Lists;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class JmxConnectionFactory {
 
-  private static final ConcurrentMap<String, AtomicInteger> SUCCESSFULL_CONNECTIONS = Maps.newConcurrentMap();
+  private static final ConcurrentMap<String, AtomicInteger> successfulConnections =
```

Comment:
nit: should be `SUCCESSFULL_CONNECTIONS`

i'll open a separate PR to add to checkstyle.xml 
```
<module name="ConstantName"/>
```
for this, so it's optional to fix here.

Reply:
see https://github.com/thelastpickle/cassandra-reaper/pull/217

Reply:
Fixed